ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/HiveInBetweenExpandRule.java

```diff
@@ -138,7 +138,7 @@ public void onMatch(RelOptRuleCall call) {
     private final RexBuilder rexBuilder;
     private boolean modified;
 
-    private RexInBetweenExpander(RexBuilder rexBuilder) {
+    RexInBetweenExpander(RexBuilder rexBuilder) {
```

Comment:
I don't see why this change is necessary. Can you explain?

Reply:
TLDR: Sorry, that's just a leftover from a previous attempt.

Long story:
At some point I attempted to expand `IN` to check the impact on simplification, but there were more side-effects than benefits, I have dropped that part but forgot to put the original visibility back.